bin/proxy.js

```diff
@@ -15,12 +15,14 @@ const log = bunyan.createLogger({
 
 const proxy = httpProxy.createProxyServer();
 const apiHost = config.get('proxyApiHost', null) || config.get('apiHost');
-const frontendHost = `http://${config.get('serverHost')}:${config.get('serverPort')}`;
+const frontendHost = `http://${config.get('serverHost')}:${config.get(
+  'serverPort'
+)}`;
```

Comment:
this is a bit weird

Reply:
Agreed. I'd rather use `oneLine` than this...